projects/packages/forms/src/blocks/field-textarea/edit.js

```diff
@@ -0,0 +1,71 @@
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import { useEffect, useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import clsx from 'clsx';
+import JetpackFieldControls from '../shared/components/jetpack-field-controls';
+import useFieldSelected from '../shared/hooks/use-field-selected';
+import useFormWrapper from '../shared/hooks/use-form-wrapper';
+import useJetpackFieldStyles from '../shared/hooks/use-jetpack-field-styles';
+import { ALLOWED_INNER_BLOCKS } from '../shared/util/constants';
+
+export default function TextareaFieldEdit( props ) {
+	const {
+		attributes,
+		clientId,
+		id,
+		isSelected,
+		label,
+		required,
+		requiredText,
+		setAttributes,
+		width,
+	} = props;
+
+	useFormWrapper( props );
+
+	const { blockStyle } = useJetpackFieldStyles( attributes );
+	const { isInnerBlockSelected, hasPlaceholder } = useFieldSelected( clientId );
+	const blockProps = useBlockProps( {
+		className: clsx( 'jetpack-field jetpack-field-textarea', {
+			'is-selected': isSelected || isInnerBlockSelected,
+			'has-placeholder': hasPlaceholder,
+		} ),
+		style: blockStyle,
+	} );
+
+	const defaultLabel = __( 'Message', 'jetpack-forms' );
+	const template = useMemo( () => {
+		return [
+			[ 'jetpack/label', { label, required, defaultLabel, requiredText } ],
+			[ 'jetpack/input', { type: 'textarea' } ],
+		];
+	}, [ label, defaultLabel, required, requiredText ] );
+
+	const innerBlocksProps = useInnerBlocksProps( blockProps, {
+		allowedBlocks: ALLOWED_INNER_BLOCKS,
+		template,
+		templateLock: 'all',
+		type: 'textarea',
```

Comment:
Is this prop right?

Reply:
Nah, I think that's a rogue leftover after trying some different approaches for the inner Input block. I'll get rid of it. Thanks.